avionics/common/src/sensor_collection.cpp

```diff
@@ -60,7 +60,7 @@ void SensorCollection::updateStatus() {
     }
     if (battery.getStatus() == SensorStatus::FAILURE) {
         LOG_WARN("Voltage sensor failed");
-        raiseToStatus(status_, RocketStatus::NONCRITICAL_FAILURE);
+        raiseToStatus(status_, RocketStatus::CRITICAL_FAILURE);
         *status_bitfield_ |= 0x04;
         // TODO: Measure voltage level before launch and if it's too low raise a critical or non-critical error
```

Comment:
Is this todo still relevant?

Reply:
Nope, ty! Will remove with the next commit